AmplifyPlugins/Core/AWSPluginsCore/Model/GraphQLDocument/Mutation/GraphQLDocument+CreateMutation.swift

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import Foundation
+
+public extension GraphQLCreateMutation {
+    convenience init(of anyModel: AnyModel,
+                     where predicate: QueryPredicate? = nil) {
+        self.init(of: anyModel.instance, where: predicate)
+    }
+}
+
+/// A concrete implementation of `GraphQLDocument` that represents a data mutation operation.
+/// The type of the operation is defined by `GraphQLMutationType`.
+public class GraphQLCreateMutation: GraphQLDocument {
+
+    public let documentType = GraphQLDocumentType.mutation
+    public let mutationType = GraphQLMutationType.create
+    public let model: Model
+    public let modelType: Model.Type
+    public let predicate: QueryPredicate?
+
+    public init(of model: Model,
+                where predicate: QueryPredicate? = nil) {
```

Comment:
I think we've agreed that there is no valid use case for predicates in Create mutations, let's remove it here